src/components/CowPenContextMenu/CowPenContextMenu.js

```diff
@@ -197,24 +197,17 @@ export const CowPenContextMenu = ({
       </TabPanel>
       <TabPanel value={currentTab} index={2}>
         <ul className="card-list">
-          <li>
-            <Item
-              {...{
-                item: cowFeed,
-                isPurchaseView: true,
-                showQuantity: true,
-              }}
-            />
-          </li>
-          <li>
-            <Item
-              {...{
-                item: huggingMachine,
-                isPurchaseView: true,
-                showQuantity: true,
-              }}
-            />
-          </li>
+          {shopInventory.map(i => (
```

Comment:
I think it's worth keeping variable names descriptive even in a limited context like this. So, let's change `i` to `item` here. As a bonus, this will let us simplify line 204 below to `item,`.